tests/test_numbers.py

```diff
@@ -251,6 +251,24 @@ def test_format_currency():
             == u'EUR 1,099.98')
     assert (numbers.format_currency(1099.98, 'EUR', locale='nl_NL')
             != numbers.format_currency(-1099.98, 'EUR', locale='nl_NL'))
+    assert (numbers.format_currency(1099.98, 'USD', format=None,
+                                    locale='en_US')
+            == u'$1,099.98')
+
+
+def test_format_currency_format_type():
```

Comment:
I'm not a big fan of using multiple assertions like that in a single test case, I would write 3 test cases: 
- `test_format_current_with_standard_format_type`
- `test_format_current_with_account_format_type`
- `test_format_current_with_invalid_format_type`

Just personal taste (this is not a blocker to approve this PR. :smile: )